src/kubernetes.h

```diff
@@ -96,6 +96,10 @@ class KubernetesReader {
       MetadataUpdater::UpdateCallback callback, const json::Object* pod,
       Timestamp collected_at, bool is_deleted) const throw(json::Exception);
 
+  // Builds the cluster full name by reading in cluster related environment
```

Comment:
`// Builds the cluster full name from cluster related environment variables.`?

Reply:
done

Reply:
I was proposing to remove `by reading` as well.

Reply:
done